tide_authenticated_content.install

```diff
@@ -132,3 +132,28 @@ function tide_authenticated_content_update_8004() {
     }
   }
 }
+
+/**
+ * Setting Exclude authenticated status processor by default.
+ */
+function tide_authenticated_content_update_8005() {
+  $config_factory = \Drupal::configFactory();
+
+  $changes = FALSE;
+  foreach ($config_factory->listAll('search_api.index.') as $index_id) {
+    $index = $config_factory->getEditable($index_id);
+    $processors = $index->get('processor_settings');
+    if (!isset($processors['authenticated_status'])) {
+      $processors['authenticated_status'] = [];
+      $index->set('processor_settings', $processors);
+      $index->save(TRUE);
+      $changes = TRUE;
+    }
+  }
+
+  if ($changes) {
+    return t('Setting Exclude authenticated status processor by default.');
```

Comment:
hey @yeniatencio 
I think, in this case, instead of using `if,else`, you probably want to use `try catch`.
and this is a drush running env, you probably want to use `dt` , not `t`

Reply:
you don't need a `return` statement.

Reply:
Hey @vincent-gao , you are right. I have updated my pr as you suggested. Thanks.